### PR TITLE
Update 'route.en.md'

### PR DESCRIPTION
**Pull request by wgy8283335**
Add some explanation of concept related to routing.

### CODE REVIEW

File: document/current/content/features/sharding/principle/route.en.md [hunk 1]
```diff
@@ -19,12 +19,36 @@ In the roadmap of ShardingSphere, elastic migration will start from 4.x version.
 
 It is used in the situation to route according to the sharding key, and can be sub-divided into 3 types, direct route, standard route and Cartesian product route.
 
+### Direct Route
 The conditions for direct route are relatively strict. 
 It requires to shard through Hint (use HintAPI to appoint the route to databases and tables directly). 
 On the premise of having database sharding but not table sharding, SQL parsing and the following result merging can be avoided. 
 Therefore, with the highest compatibility, it can execute any SQL in complex situations including sub-queries, self-defined functions. 
-Direct route can also be used in the situation where sharding keys are not in SQL.
+Direct route can also be used in the situation where sharding keys are not in SQL.For example, set sharding key to '3'.
```
Comment (terrymanu) [inline document/current/content/features/sharding/principle/route.en.md]: Add space after `.`

Comment (terrymanu) [inline document/current/content/features/sharding/principle/route.en.md]: use `` as mark instead of ''

Comment (wgy8283335) [reply document/current/content/features/sharding/principle/route.en.md]: THX. I've modified the document.

Comment (wgy8283335) [reply document/current/content/features/sharding/principle/route.en.md]: THX. I've modified the document.

Comment (wgy8283335) [reply document/current/content/features/sharding/principle/route.en.md]: @terrymanu. Should I create another pull request and close this one.

**Comment by terrymanu** [comment]
This pr is fine, I just merge it

**Pull request merged**